build/vs2017.bat

```diff
@@ -1,2 +1,2 @@
 call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat"
-msbuild SonarLint.VisualStudio.Integration.sln /t:rebuild /p:configuration=Release /p:DeployExtension=false /p:ZipPackageCompressionLevel=normal /v:m /p:defineConstants=SignAssembly /p:SignAssembly=true /p:AssemblyOriginatorKeyFile=%CERT_PATH%
\ No newline at end of file
+msbuild SonarLint.VisualStudio.Integration.sln /t:rebuild /p:configuration=Release /p:DeployExtension=false /p:ZipPackageCompressionLevel=normal /v:m /p:defineConstants=SignAssembly /p:SignAssembly=true /p:AssemblyOriginatorKeyFile=%CERT_PATH% /p:VsTargetVersion=%vsTargetVersion%
```

Comment:
The MSBuild targets are expecting 2015, 2017 or 2019, not VS2015 etc.
I think it's ok just to have 2015 etc since the variable name is `vsTargetVersion`.